hack/test-cmd.sh

```diff
@@ -55,6 +56,11 @@ tests=( $(find_tests ${1:-.*}) )
 
 # Setup environment
 
+if [[ -z "${JUNIT_REPORT_OUTPUT+x}" ]]; then
+  # if no jUnit report output file is set, throw away our jUnit info
+  JUNIT_REPORT_OUTPUT="/dev/null"
```

Comment:
@deads2k the jUnit functions need `$JUNIT_REPORT_OUTPUT` to be set. I don't think it's reasonable to stick a check in every single function, so instead an `if` case like this in any script using jUnit is more appropriate.